usage/camp/src/test/java/org/apache/brooklyn/camp/brooklyn/DslAndRebindYamlTest.java

```diff
@@ -124,6 +126,19 @@ public void testDslAttributeWhenReady() throws Exception {
     }
 
     @Test
+    public void testDslAttributeWhenReadyPersisted() throws Exception {
```

Comment:
I set up a test with that (see below), except that I now get `java.io.NotSerializableException: org.apache.brooklyn.entity.group.DynamicClusterImpl$NextClusterMemberIdSupplier`
Will look into it.